source/styles/index.css

```diff
@@ -101,7 +101,7 @@ button:hover {
     text-align: center;
 }
 
-#cancelButton, #subButton, #cancelEditButton, #subEditButton, .deleteEditButton  {
+#cancelButton, #subButton, #cancelEditButton, #subEditButton, .deleteEditButton, .submitEditButton {
```

Comment:
If `#subEditButton` is not needed in the javascript and removed in the index file, it should also be removed here.

Reply:
Fixed